deployment/stacks/api.ts

```diff
@@ -52,7 +52,10 @@ export function createApiComponents(props: CreateApiComponentsProps) {
   Tags.of(scope).add("SERVICE", service);
 
   const vpc = ec2.Vpc.fromLookup(scope, "Vpc", { vpcName });
-  const kafkaAuthorizedSubnets = getSubnets(scope, kafkaAuthorizedSubnetIds)
+  const kafkaAuthorizedSubnets = getSubnets(
+    scope,
+    kafkaAuthorizedSubnetIds ?? ""
```

Comment:
It this still necessary? I think it will always be there. Oh is this because of the dang bootstrap thing. Ugh. That's cool.

Reply:
yeah, the bootstrap :(